src/cli/node/json_rpc_client.rs

```diff
@@ -242,6 +242,66 @@ pub fn get_reputation(
     Ok(())
 }
 
+pub fn get_miners(addr: SocketAddr, init: i64, end: i64, csv: bool) -> Result<(), failure::Error> {
+    let mut stream = start_client(addr)?;
+    let params = GetBlockChainParams {
+        epoch: init,
+        limit: end,
+    };
+    let response = send_request(
+        &mut stream,
+        &format!(
+            r#"{{"jsonrpc": "2.0","method": "getBlockChain", "params": {}, "id": 1}}"#,
+            serde_json::to_string(&params).unwrap()
+        ),
+    )?;
+    log::info!("{}", response);
+    let block_chain: ResponseBlockChain<'_> = parse_response(&response)?;
+    let mut hm = HashMap::new();
+
+    if csv {
+        println!("Block number;Block hash;Miner hash")
+    } else {
+        println!("Blockchain:");
+    }
+    for (epoch, hash) in block_chain {
+        let request = format!(
+            r#"{{"jsonrpc": "2.0","method": "getMiner", "params": [{:?}], "id": "1"}}"#,
```

Comment:
I would prefer to avoid adding a new JSON-RPC method just for this, you can easily implement the same functionality in the CLI by calling getBlock and doing

```rust
let block: Block = parse_response(&response)?;
let miner_hash = block.block_sig.public_key.pkh();
```

```suggestion
            r#"{{"jsonrpc": "2.0","method": "getBlock", "params": [{:?}], "id": "1"}}"#,
```